pcdsdevices/ipm.py

```diff
@@ -136,6 +137,51 @@ def remove(self, moved_cb=None, timeout=None, wait=False):
             return (rmstatus & self.diode.remove(moved_cb=moved_cb,
                                                  timeout=timeout, wait=wait))
 
+    def target_in(self, target_num, moved_cb=None, timeout=None, wait=False):
+        """
+        Moves the target to one of the target positions. There are 4 targets
+        with different thinkness and absorbtion/signal.
```

Comment:
```suggestion
        with different thickness and absorption/signal.
```

Reply:
thanks for the corrections, I need to get better at catching my misspelled comments - cause I do this all the time :( 
do you know of any plugins that I can install that checks for grammar :D ?

Reply:
It's hard to do this generically for code... I don't have a spellcheck plugin and I also make these kinds of typos all the time, especially in docstrings where I don't read it very closely again after writing it once.